src/Support/Fakery/Stub.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace RicorocksDigitalAgency\Soap\Support\Fakery;
+
+use Closure;
+use Illuminate\Support\Str;
+use RicorocksDigitalAgency\Soap\Request\Request;
+
+class Stub
+{
+    public $endpoint;
+    public $methods;
+    public $callback;
+
+    public static function for($endpoint): self
+    {
+        $instance = app(self::class);
+        $instance->register($endpoint);
+
+        return $instance;
+    }
+
+    public function respondWith($callback): self
+    {
+        $this->callback = $callback;
+
+        return $this;
+    }
+
+    public function getResponse(Request $request)
+    {
+        return $this->callback instanceof Closure ? call_user_func($this->callback, $request) : $this->callback;
```

Comment:
Can we drop the ternaries on a new line?

Reply:
Indeed we can! Thanks for the spot 👍